pkg/vrct/vrctFs/fs_methods.go

```diff
@@ -8,44 +8,37 @@ import (
 	"strings"
 )
 
-func (v *FsVRCT) CreateFile(filePath string, content []byte, isOptional bool) error {
+// CreateFile TODO: check for conflict out of the box
+func (v *FsVRCT) CreateFile(filePath string, content []byte, isOptional bool, fileType int) error {
+	// TODO: allow non-absolute paths
 	filePath, err := pathMustBeAbsolute(filePath)
```

Comment:
I think we can easily change this code line and convert a relative path to the absolute path using the `filepath` module like this.
```suggestion
path, err := filepath.Abs(path)
```

Reply:
I didn't know about this, thx